psql2mysql/tests/test_dbwrapper.py

```diff
@@ -82,3 +82,37 @@ def test_scanTableForLongTexts(self, mock_query_long_text_rows,
         wrong_item = result[0]
         self.assertEqual("text", wrong_item["column"])
         self.assertEqual("id=3", wrong_item["primary"][0])
+
+    def test_chunked_write_table_rows(self):
+        def reset_mocks():
+            mock_connection = mock.MagicMock()
+            self.db_wrapper.connection = mock_connection
+            mock_rows = mock.MagicMock()
+            mock_rows.fetchmany.side_effect = [['one response'],
+                                               ['second response'], []]
+            mock_rows.fetchall.side_effect = [['one response'], ]
+            return mock_rows, mock_connection
+
+        mock_table = mock.MagicMock()
+
```

Comment:
These two tests check different things. I'd rather they were two separate test cases, one checking fetchall, and the other doing fetchmany.